app/routes/_index.tsx

```diff
@@ -1,16 +1,102 @@
 import type { V2_MetaFunction } from '@remix-run/react';
 import HeroSection from '~/components/hero-section';
-import EventsSection from '../components/events-section';
+import { EventsSection } from '../components/events-section';
 
 export const meta: V2_MetaFunction = () => {
   return [{ title: 'Plan It Social' }];
 };
 
+type Event = {
```

Comment:
In the last tutoring session, we defined the event schema in the `prisma.schema` file. Prisma generates TypeScript types from these schemas, which we can use in our code. Maybe you can use the auto-generated types here instead?

`import type { Event, Group, User } from '@prisma/client';`

Reply:
I think using Prisma types is a good idea. Could we include imgUrl and imgAlt in the Event model in schema.prisma? Also, should we add a 'location' column to the model if we plan on having events in different cities or countries?

Reply:
Oh yeah for sure! :) Feel free to add those fields!